llvm/include/llvm/IR/Instruction.h

```diff
@@ -86,6 +86,11 @@ class Instruction : public User,
   /// Returns true if any DPValues are attached to this instruction.
   bool hasDbgValues() const;
 
+  /// Transfer any DPValues on the position \p It onto this instruction.
+  /// \returns true if adoption worked, false otherwise.
+  bool adoptDbgValues(BasicBlock *BB, InstListType::iterator It,
```

Comment:
Is there a way to word this where the method of moving is more explicitly called out (plus consequences of returning true/false, if it's easy to summarise)?